xapian-letor/m4-macros/xapianletor.m4

```diff
@@ -0,0 +1,172 @@
+# Get XAPIAN_CXXFLAGS, XAPIAN_LIBS, and XAPIAN_VERSION from xapianletor-config and
```

Comment:
This really needs to use a different prefix to xapian-core or else things won't work if the user tries to use both libraries together (and I'd expect almost anything that use xapian-letor would need to use the core API as well).

Reply:
fixed in a fixup commit.